composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
Libraries should never have .lock files. But this is an interesting point. I'll think about this a little.